generators/server/templates/src/main/java/package/config/SecurityConfiguration_reactive.java.ejs

```diff
@@ -425,7 +432,7 @@ public class SecurityConfiguration {
                     return Mono.just(jwt);
                 }
                 // Retrieve user info from OAuth provider if not already loaded
-                return users.computeIfAbsent(jwt.getSubject(), s -> {
+                return users.get(jwt.getSubject(), s -> {
```

Comment:
Does this method work the same as the previous one did? That is, it checks the cache and if it doesn't exist, then it does the lookup.

Reply:
Yes. From the doc:

`Returns the value associated with the key in this cache, obtaining that value from the mappingFunction if necessary. This method provides a simple substitute for the conventional "if cached, return; otherwise create, cache and return" pattern.`